src/views/router_views/choose_your_package_accounts/choose_your_package_accounts.njk

```diff
@@ -5,24 +5,33 @@
 
 {% block main_content %}
 
-  {{ 
-    govukRadios({
-      name: "PackageAccounts",
-      fieldset: {
-        legend: {
-          text: title,
-          isPageHeading: true,
-          classes: "govuk-fieldset__legend--l"
-        }
-      },
-      items: packageAccountsItems
-    }) 
-  }}
+    {% set errorMessage = undefined %}
+    {% if errors[packageTypeFieldName] %}
+        {% set errorMessage = { text: errors[packageTypeFieldName].summary } %}
+    {% endif %}
 
-  {{ 
-    govukButton({
-      text: "Confirm and continue"
-    }) 
-  }} 
+  <form  method="post">
+    {{ 
+      govukRadios({
+        name: packageTypeFieldName,
+        fieldset: {
+          legend: {
+            text: title,
+            isPageHeading: true,
+            classes: "govuk-fieldset__legend--l"
+          }
+        },
+        errorMessage: errorMessage,
+        items: packageAccountsItems
+      }) 
+    }}
 
-{% endblock %}
\ No newline at end of file
+    {{ 
+      govukButton({
+        text: "Confirm and continue"
```

Comment:
Do you not need "type : submit" here, or is that automatic?

Reply:
type: "submit" is the default for buttons in forms.
 https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#type
 
 However, if you think it would be better to be explict, I can add it.

Reply:
It's ok leave it as is